pom.xml

```diff
@@ -8,7 +8,7 @@
     </parent>
     <groupId>org.webjars</groupId>
     <artifactId>bootstrap-table</artifactId>
-    <version>1.15.6-SNAPSHOT</version>
+    <version>1.16.0</version>
```

Comment:
This needs to be `1.16.0-SNAPSHOT` because Maven performs the actual release.

Reply:
ok，updated!